project/build.scala

```diff
@@ -18,13 +19,18 @@ object ContentApiClientBuild extends Build {
     base = file(".")
   )
   .enablePlugins(BuildInfoPlugin)
+  .settings(ScroogeSBT.newSettings: _*)
   .settings(
-      buildInfoKeys := Seq[BuildInfoKey](version),
-      buildInfoPackage := "com.gu.contentapi.buildinfo"
+    buildInfoKeys := Seq[BuildInfoKey](version),
+
+    buildInfoPackage := "com.gu.contentapi.buildinfo",
+    buildInfoObject := "CapiBuildInfo"
   )
   .settings(releaseSettings: _*)
   .settings(sonatypeSettings: _*)
   .settings(
+    unmanagedJars in Compile := (baseDirectory.value / "lib" ** "*.jar").classpath,
```

Comment:
I think I prefer that its explicit and you can see that there are unmanaged jars being used in the project, but if removing it is preferred then I'm fine with that?